lerobot/common/policies/act/configuration_act.py

```diff
@@ -139,7 +137,7 @@ class ACTConfig:
     n_vae_encoder_layers: int = 4
 
     # Inference.
-    temporal_ensemble_momentum: float | None = None
+    temporal_ensemble_coeff: float | None = None
```

Comment:
It would be nice to have a comment after this like # the value used in ACT is 0.01, for details see ACTTemporalEnsembler